x-pack/plugins/security/public/views/logged_out/logged_out.tsx

```diff
@@ -22,7 +23,18 @@ chrome
       const domNode = document.getElementById('reactLoggedOutRoot');
       render(
         <I18nContext>
-          <LoggedOutPage addBasePath={chrome.addBasePath} />
+          <AuthenticationState
```

Comment:
__note:__ I defined this entire tree here, but let me know if you prefer it to be in a separate file.